resources/payload-templates/dropper.cs

```diff
@@ -100,6 +103,11 @@ static byte[] Combine(byte[] first, byte[] second)
 	}
 	static System.Net.WebClient GetWebRequest(string cookie)
 	{
+		try {
+			ServicePointManager.SecurityProtocol = (SecurityProtocolType)192 |(SecurityProtocolType)768 | (SecurityProtocolType)3072;			
```

Comment:
Does this fix the TLS issues?

Reply:
Yeah, if a device has v4.0 and v4.5 frameworks installed you can define the value of the types directly and thus support TLSv1.2 on v4.0 payloads. Didn't realise Ben had already put this fix into public, you snooze you lose :)

If v4.5 doesn't exist on the host though, it'll crash, hence the try catch.

Reply:
Hopefully - need to test a lot to make sure it doesnt break anything else, e.g. dotnet v4.0 etc but the try catch should save that I think

Reply:
Ah sweet that's awesome :D

Reply:
@PwnDexter may want to update your FindFrontableDomains fork's colouring if so